### PR TITLE
Fix regression caused by ClassId conversion to class name vs hexid

### PR DESCRIPTION
**Pull request by khanaffan**
itwinjs-core: https://github.com/iTwin/itwinjs-core/pull/7238

### CODE REVIEW

**Review by rschili — APPROVED** [review]

File: iModelJsNodeAddon/api_package/ts/src/test/ECSqlStatement.test.ts [hunk 1]
```diff
@@ -7,11 +7,10 @@ import { assert, expect } from "chai";
 import * as fs from "fs-extra";
 import * as path from "path";
 import { DbResult, Guid } from "@itwin/core-bentley";
-import { QueryRowFormat } from "@itwin/core-common";
 import { IModelJsNative } from "../NativeLibrary";
 import { getOutputDir, iModelJsNative } from "./utils";
 
-describe("ECSqlStatement", () => {
+describe.only("ECSqlStatement", () => {
```
Comment (nick4598) [inline iModelJsNodeAddon/api_package/ts/src/test/ECSqlStatement.test.ts:13]:
```suggestion
describe("ECSqlStatement", () => {
```

Comment (khanaffan) [reply iModelJsNodeAddon/api_package/ts/src/test/ECSqlStatement.test.ts]: Removed.

**Comment by aruniverse** [comment]
@mergifyio backport release/4.9.x

**Comment by aruniverse** [comment]
@pmconne mind reviewing this? [latest pipeline run](https://dev.azure.com/bentleycs/iModelTechnologies/_build/results?buildId=3262088&view=results) should succeed any min now

**Comment by pmconne** [comment]
> @pmconne mind reviewing this? [latest pipeline run](https://dev.azure.com/bentleycs/iModelTechnologies/_build/results?buildId=3262088&view=results) should succeed any min now

Waiting for resolution of questions raised on itwinjs-core PR.

**Review by pmconne — APPROVED** [review]

**Pull request merged**